.github/workflows/main.yml

```diff
@@ -16,35 +16,32 @@ jobs:
     strategy:
       matrix:
         PEXT_BUILD_PORTABLE: [0, 1]
-    name: "AppImage (portable: ${{ matrix.PEXT_BUILD_PORTABLE }})"
-    runs-on: ubuntu-18.04
+        # Operative systems: [ubuntu-18.04, ubuntu-20.04, ubuntu-22.04]
+        # The Ubuntu version 22.04 has problems with Linux script to deploy plugin conda.
+        OPERATIVE_SYSTEM: [ubuntu-latest]
```

Comment:
I agree. I don't have experience with AppImage, also I didn't understand how to use this build files in my Linux to use the app. You are the expert here. Is it fine to change the version to ubuntu-18.04?

Reply:
Yeah, 18.04 is the oldest still supported Ubuntu release so that seems like the best option (that way security patches do make it into the AppImage)

Reply:
Only be aware if the GitHub action is failing because Ubuntu 18.04 is deprecated on 12/1/22. If it fails you need to move to the latest (stable version) in GitHub which is the 20.04.

[GitHub Actions ➟ Supported runners and hardware resources](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)

[GitHub Actions: The Ubuntu 18.04 Actions runner image is being deprecated and will be removed by 12/1/22](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/)

```text
Ubuntu 18.04 [deprecated]
ubuntu-18.04
Migrate to ubuntu-20.04 or ubuntu-22.04. For more information, see this GitHub blog post.
```

Same for macOS 10.15:

[GitHub Actions The macOS 10.15 Actions runner image is being deprecated and will be removed by 8/30/22](https://github.blog/changelog/2022-07-20-github-actions-the-macos-10-15-actions-runner-image-is-being-deprecated-and-will-be-removed-by-8-30-22/)

```text
macOS Catalina 10.15 [deprecated]
macos-10.15
Migrate to macOS-11 or macOS-12. For more information, see this GitHub blog post.
```

Reply:
Oh wow, what an annoying thing for GitHub to do, but I kinda get it.

For macOS I don't think we need to build on an older version, AppImage is a Linux specific thing. I assume that building on the latest macOS should just work for all (hopefully)

Reply:
Solved in the commit: 9391904.

All the GitHub actions are green and the artifacts in place:
- https://github.com/airvzxf/Pext/actions/runs/3324538000
- https://github.com/Pext/Pext/actions/runs/3324538211